README.md

```diff
@@ -1,15 +1,39 @@
 # express-pino-logger&nbsp;&nbsp;[![Build Status](https://travis-ci.org/mcollina/express-pino-logger.svg)](https://travis-ci.org/mcollina/express-pino-logger)
 
 An [express](http://npm.im/express) middleware to log with
-[pino](https://github.com/mcollina/pino).
+[pino](https://github.com/mcollina/pino). Incidentally, it also works
+without express.
 
-From our benchmarks, this is 4x faster than
-[express-bunyan-logger](http://npm.im/express-bunyan-logger) and 25%
-faster than [winston-express](http://npm.im/winston-express).
-It is slightly slower (< 5%) than [morgan](http://npm.im/morgan),
-which is a usecase-specific logger. With
-[extreme](https://github.com/mcollina/pino#extreme) mode enable, it is
-faster than morgan.
+From our [benchmarks](#benchmarks), it is the fastest logger in town for
+any [express](http://npm.im/express) application.
+
+* [Installation](#install)
+* [Usage](#usage)
+* [Benchmarks](#benchmarks)
+* [API](#api)
+* [Team](#team)
+* [Acknowledgements](#acknowledgements)
+* [License](#license)
+
+## Benchmarks
+
+These are taken logging each request/response pair while returning
+`'hello world'`, using
+[autocannon](https://github.com/mcollina/autocannon) with 100
+connections and 10 pipelined requests.
+
+* `express-bunyan-logger`: 1950 req/sec
+* `express-winston`: 4739 req/sec
+* `morgan`: 5812 req/sec
+* `express-pino-logger`: 6108 req/sec
+* `express-pino-logger` (extreme): 6594 req/sec
+* `express-pino-logger` (without express and extreme): 11988 req/sec
+
+It is important to note that [morgan](http://npm.im/morgan) is a
+specialized logger for express and HTTP data. Also, morgan does not log
+in JSON, and it does not support aribitrary data structures.
```

Comment:
While we're comparing `express-pino-logger` against [morgan](http://npm.im/morgan), this isn't really a fair contest. Morgan logs aren't in JSON format, nor does it support arbitrary data . Further Morgan uses a form of `eval` to achieve high speed logging (something that may put a lot of developers off). 

The fact that `express-pino-logger` achieves higher throughput with JSON logging and and arbitrary data, and without using `eval` testifies to its' high performance profile.